src/main/java/core/basesyntax/FileEditor.java

```
@@ -0,0 +1,187 @@
+                infoMessage();
+            } else {
+                switch (variant[0].toLowerCase()) {
+                    case "create":
+                        if (variant.length == 3) {
+                            create(variant[1], variant[2]);
+                        } else {
+                            infoMessage();
+                        }
+                        break;
+                    case "read":
+                        if (variant.length == 3) {
+                            read(variant[1], variant[2]);
+                        } else {
+                            infoMessage();
+                        }
+                        break;
+                    case "info":
+                        if (variant.length == 3) {
+                            info(variant[1], variant[2]);
+                        } else {
+                            infoMessage();
+                        }
+                        break;
+                    case "help":
+                        if (variant.length == 2) {
+                            help(variant[1]);
+                        } else {
+                            help();
+                        }
+                        break;
+                    case "exit":
+                        exit();
+                        break;
+                    default:
+                        wantToSave();
+                }
+            }
+
+        }
+    }
+
+    private void infoMessage() {
+        System.out.println("Please enter valid command with correct amount parameters.\n"
+                + "You can get list of available commands by entering 'help'.\n");
+    }
+
+    private void create(String path, String filename) {
+        try {
+            Path newFilePath = Paths.get(path + FileSystems.getDefault().getSeparator() + filename);
+            Files.createFile(newFilePath);
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
```

better to throw an exception